contrib/builder/rpm/opensuse-12.3/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#
+# THIS FILE IS AUTOGENERATED; SEE "contrib/builder/rpm/generate.sh"!
+#
+
+FROM jess/opensuse:12.3
```

Comment:
Is 12.3 still active enough to warrant asking the image maintainer if he's willing to add it?